ui-core/src/types/landingPageConfig.ts

```diff
@@ -50,25 +50,28 @@ export type NavbarItem =
   | NavbarItemMailingList
   | NavbarItemExternal
 
-export type NavbarItemInternal = {
-  itemType: 'INTERNAL'
+export type BaseNavBarItem = {
```

Comment:
Suggest not exporting this to avoid the temptation to use it elsewhere. Elsewhere, you almost always want a union of the different NavBarItem types vs this base type.